src/Maui/Pages/Combat.razor

```diff
@@ -1,83 +1,73 @@
-                    <div class="d-flex flex-column align-center">
-                        @if (selected != null)
-                        {
-                     @if (selected.Text == "Actions")
-                      {
-                                    @foreach (Data.Action action in activeCreature.Creature.Actions)
-                                {
-                                    <MudText>@action.Name</MudText>
-                                    @DisplayAction(action.Desc)
-                            }
-                        }
-                        else if(selected.Text == "Abilities")
-                        {
-                                    @foreach (Data.Ability ability in activeCreature.Creature.Abilities)
-                                {
-                                <MudText>@ability.Name</MudText>
-                                <MudText>@ability.Desc</MudText>
-                            }                   
-                        }
-                        }
+            <MudText>@action.Name</MudText>
+            @DisplayAction(action.Desc)
+        }
+        <MudButton Text="Close" Color="Color.Error" OnClick="@(() => CloseDrawer())">Close</MudButton>
+    </MudDrawer>
 
-                      </div>
-            <MudStack Row="true">
-                            <MudPaper Class="pa-3">STR: @activeCreature.Creature.StrPoints (@DiceRoller.CalculateAbilityScoreModifier(activeCreature.Creature.StrPoints))</MudPaper>
-                            <MudPaper Class="pa-3">DEX: @activeCreature.Creature.DexPoints (@DiceRoller.CalculateAbilityScoreModifier(activeCreature.Creature.ConPoints))</MudPaper>
-                            <MudPaper Class="pa-3">WIS: @activeCreature.Creature.WisPoints (@DiceRoller.CalculateAbilityScoreModifier(activeCreature.Creature.WisPoints))</MudPaper>
-                            <MudPaper Class="pa-3">INT: @activeCreature.Creature.IntPoints (@DiceRoller.CalculateAbilityScoreModifier(activeCreature.Creature.IntPoints))</MudPaper>
-                            <MudPaper Class="pa-3">CHA: @activeCreature.Creature.ChaPoints (@DiceRoller.CalculateAbilityScoreModifier(activeCreature.Creature.ChaPoints))</MudPaper>
-            </MudStack>
-            </ChildContent>
-                </MudExpansionPanel>
-</MudExpansionPanels>
-}
-    </MudItem>
 </MudGrid>
 
+
+@using DMPowerTools.DataModels;
+@using DMPowerTools.Maui.DataModels;
 @code {
     @page "/combat"
-    @using DMPowerTools.DataModels;
-    @using DMPowerTools.Maui.DataModels;
-    @using DMPowerTools.Maui.Data;
     @using Microsoft.EntityFrameworkCore;
     @using Markdig;
 
     @inject ApplicationDbContext DBContext;
     string returnCreature { get; set; }
     IEnumerable<Creature> creatures;
     List<InitiatedCreature> InitiatedCreatures = new();
+    InitiatedCreature clickedCreature = new();
     Creature selectedCreature = new();
     MudChip selected;
+    bool open;
+    Anchor anchor;
```

Comment:
What is this?

Reply:
anchor isnt needed now, but the open is the trigger for the drawer